core/settings/qPrefProxy.h

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: GPL-2.0
+#ifndef QPREFPROXY_H
+#define QPREFPROXY_H
+#include "core/pref.h"
+
+#include <QObject>
+
+
+class qPrefProxy : public QObject {
+	Q_OBJECT
+	Q_PROPERTY(bool proxy_auth READ proxy_auth WRITE set_proxy_auth NOTIFY proxy_auth_changed);
+	Q_PROPERTY(QString proxy_host READ proxy_host WRITE set_proxy_host NOTIFY proxy_host_changed);
+	Q_PROPERTY(QString proxy_pass READ proxy_pass WRITE set_proxy_pass NOTIFY proxy_pass_changed);
+	Q_PROPERTY(int proxy_port READ proxy_port WRITE set_proxy_port NOTIFY proxy_port_changed);
+	Q_PROPERTY(int proxy_type READ proxy_type WRITE set_proxy_type NOTIFY proxy_type_changed);
+	Q_PROPERTY(QString proxy_user READ proxy_user WRITE set_proxy_user NOTIFY proxy_user_changed);
+
+public:
+	qPrefProxy(QObject *parent = NULL);
+	static qPrefProxy *instance();
+
+	// Load/Sync local settings (disk) and struct preference
+	void loadSync(bool doSync);
+	void inline load() { loadSync(false); }
+	void inline sync() { loadSync(true); };
```

Comment:
As we all know, thanks to @bstoeger , that second `;` after the `}` isn't needed and should be dropped for consistency...

Reply:
done